src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -57,6 +57,15 @@ internal TagHelperExecutionContext(string tagName)
         }
 
         /// <summary>
+        /// Indicates whether or not the tag is self closing.
+        /// </summary>
+        public bool SelfClosing
```

Comment:
@NTaylorMullen
3. `TagHelperExecutionContext` has an `Output` property of type `TagHelperOutput`